gstreamer/tests/test_media_manager.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+from gstreamer.media_manager import *
```

Comment:
Using a wildcard is generally a bad practice. Only import GstManager:
`from gstreamer.media_manager import GstManager`